js/modules/k6/k6_test.go

```diff
@@ -182,7 +225,16 @@ func checkTestRuntime(t testing.TB, ctxs ...*context.Context) (
 	ctx = common.WithRuntime(ctx, rt)
 	ctx = lib.WithState(ctx, state)
 	state.BuiltinMetrics = metrics.RegisterBuiltinMetrics(metrics.NewRegistry())
-	require.NoError(t, rt.Set("k6", common.Bind(rt, New(), &ctx)))
+	m, ok := New().NewModuleInstance(
+		&modulestest.VU{
+			RuntimeField: rt,
+			InitEnvField: &common.InitEnvironment{},
+			CtxField:     common.WithRuntime(ctx, rt),
```

Comment:
Here `common.WithRuntime` is called twice: at L225 and at L232. Shouldn't it be twice?